app/javascript/controllers/actors_controller.js

```diff
@@ -0,0 +1,40 @@
+import { Controller } from 'stimulus'
+import axios from 'axios'
+import { csrfToken, jumpToCreator } from './stimulus_modules'
+
+export default class extends Controller {
+  // Intercept Rails UJS's ajax:success callback and update the modal with the resulting content.
+  // This avoids having to have a separate .js.erb partial in a typical Rails AJAX setup.
+  success (event) {
+    const [data, status, xhr] = event.detail // eslint-disable-line no-unused-vars
+    document.getElementsByClassName('modal-content').item(0).innerHTML = data.body.innerHTML
```

Comment:
Even though there's only one modal in the layout (currently!), it would be more robust and more idiomatic javascript if the modal-content element had an ID, and this line here used `getElementById`. It's a good habit to try to use IDs and not classes whenever searching for elements in js (sometimes lookup by class is totally warranted, but much of the time it's not a good idea)